src/domain/hooks/hooks.repository.ts

```diff
@@ -73,12 +77,28 @@ export class HooksRepositoryWithNotifications implements IHooksRepository {
   }
 
   async onEvent(event: Event): Promise<unknown> {
-    return Promise.allSettled([
-      this.onEventClearCache(event),
-      this.eventNotificationsHelper.onEventEnqueueNotifications(event),
-    ]).finally(() => {
-      this.onEventLog(event);
-    });
+    const isSupportedChainId = await this.isSupportedChainMemo(event.chainId);
+    if (isSupportedChainId) {
+      return Promise.allSettled([
+        this.onEventClearCache(event),
+        this.eventNotificationsHelper.onEventEnqueueNotifications(event),
+      ]).finally(() => {
+        this.onEventLog(event);
+      });
+    } else {
+      this.loggingService.warn({
+        type: 'unsupported_event_chain',
```

Comment:
If this follows a naming convention we can leave it as is, but `unsupported_chain_event` reads better (also would need changing in the other service).

Reply:
Totally agree! Changed in [789cee5](https://github.com/safe-global/safe-client-gateway/pull/1920/commits/789cee533294c8d1e6717e74004882002c737b01)